xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.adm.sudo"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>sudo <!-- TODO: Better title --></title>
+  <subtitle><quote>With great power comes great responsibility</quote></subtitle>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker></dm:bugtracker>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+
+  <para>
+    Many commands and system utilities need to be run as &rootuser;
+    to modify files and/or perform tasks that only the super user is allowed to.
+
+    For security purposes and to avoid running dangerous commands accidentially,
+    it is generally disregarded to log in as &rootuser; directly.
+    Instead, it is recommended to work as a normal, unprivileged, user and
+    use the &sudo; command to run commands with elevated privileges.
+  </para>
+
+  <para>
+    In its default configuration, &sudo; is, apart of different options,
+    similiar to <command>su</command>.
+    Compared to <command>su</command>, &sudo; offers the possibility to allow users
+    to run commands with privileges of any other user in a highly configurable manner.
+    While <command>su</command> always uses PAM for authenticating as target user,
+    &sudo; makes it possible to authenticate with your own credentials.
+    This increases security by not having to share the &rootuser; password.
+
+    For instance, you can allow members of the group &examplegroup; to run a command <command>frobnicate</command>
+    as &exampleuserII;, with the restriction that no arguments are specified.
+    This can be used to assign roles with specific abilities to certain users and groups.
+  </para>
```

Comment:
~~For instance, you can allow members of the group &examplegroup; to run a command <command>frobnicate</command>
 as &exampleuserII;, with the restriction that no arguments are specified.
 This can be used to assign roles with specific abilities to certain users and groups.~~